### PR TITLE
Cosmetics - fix dev container

### PR DESCRIPTION
**Pull request by AnotherDaniel**

### CODE REVIEW

File: .github/workflows/README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ This is implemented in [`nightly.yaml`](nightly.yaml)
 
 In addition to the main workflows described above, there exist a number of modules that are used by these main workflows. They can also be run standalone, and are intendet to make composing the capabilities of our main workflows simpler. These are:
 
+- `verify-msrv.yaml` - checks if the MSRV ('Minimum Supported Rust Version) declared in Cargo.toml is correct
```
[the remark this answers is not in the document]
Comment (AnotherDaniel) [reply .github/workflows/README.md]: move to separate PR

**Comment by AnotherDaniel** [comment]
> I am not sure why there are any changes regarding the `up-spec` folder in the second commit. FMPOV these should not be included. In any case, FMPOV this PR should be split up in two: one dealing with the CI workflows and the other with the devcontainer. The devcontainer fixes are currently split between the two commits.

Doing that

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -52,7 +50,7 @@
             ]
         }
     },
-    "workspaceMount": "source=${localWorkspaceFolder},target=/workspace/uprotocol-rust/,type=bind",
-    "workspaceFolder": "/workspace/uprotocol-rust/",
+    "workspaceMount": "source=${localWorkspaceFolder},target=/workspace/up-rust/,type=bind",
+    "workspaceFolder": "/workspace/up-rust/",
```
[the remark this answers is not in the document]
Comment (AnotherDaniel) [reply .devcontainer/devcontainer.json:54]: The json linter is auto-removing that on save, so it doesn't seem to be a thing for json?

**Comment by AnotherDaniel** [comment]
> I am not sure why there are any changes regarding the `up-spec` folder in the second commit. FMPOV these should not be included. 

Those were a mistake - removed.

File: .devcontainer/devcontainer.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (sophokles73) [reply .devcontainer/devcontainer.json:54]: It does not have to do with the content type but is more a best practice when developing across different platforms, preventing different line ends to cause Git to believe a file has changed ...

[the remark this answers is not in the document]
Comment (sophokles73) [reply .devcontainer/devcontainer.json:54]: never mind

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**